SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -36,7 +36,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bBatchSpatialPositionUpdates(false)
 	, MaxDynamicallyAttachedSubobjectsPerClass(3)
 	, bEnableServerQBI(true)
```

Comment:
Now that we warn if this flag is on, we should probably disable by default

Reply:
Seems like a pretty big unrelated change, but can add if you want

Reply:
Have raised a ticket to do this instead of as part of this: https://improbableio.atlassian.net/browse/UNR-2827